src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -288,9 +288,81 @@ boolean canAddField() {
       }
     }
   }
+
+  /**
+   * Filters db documents based on non document id field values if id use "$in" operator, use
```

Comment:
maybe "based on values of fields other than document id: for filtering on id field use IDFilter..."?

Reply:
sure!